.github/workflows/trivy.yml

```diff
@@ -87,9 +152,15 @@ jobs:
           sed -i 's/format: sarif/format: table/g' utils/trivy/trivy.yaml
           sed -i 's/exit-code: 0/exit-code: 1/g' utils/trivy/trivy.yaml
 
-      - name: Run Trivy vulnerability scanner in filesystem mode
-        uses: aquasecurity/trivy-action@6e7b7d1fd3e4fef0c5fa8cce1229c54b2c9bd0d8  # 0.24.0
+      - name: Run Trivy vulnerability scanner in filesystem mode (human readable format)
+        # master to have 'skip-setup-trivy'
+        uses: aquasecurity/trivy-action@d2a392a13760cb64cb6bbd31d4bed2a7d9a5298d
         with:
           scan-type: 'fs'
           scan-ref: '.'
           trivy-config: 'utils/trivy/trivy.yaml'
+          skip-setup-trivy: true
+          cache: ${{ steps.db-downloaded.outputs.cache-hit == 'true' }}
```

Comment:
Again, something that should always be `true` given the prerequisite job?

```suggestion
          cache: true
```

Reply:
We intentionally avoid using cache when cache is not available in the default branch.